.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 # https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
 defaults: &defaults
   docker:
-    - image: quay.io/cortexproject/build-image:validate-k8s-specs-7c217ee7
+    - image: quay.io/cortexproject/cortex:update-gogoproto-6e528d554
```

Comment:
Note to both: rebuild the image once the PR is ready to merge and replace the tag here.

Reply:
I think we'll keep this one.

Reply:
welp it seems that this is not a build image. @gouthamve can you help me here please. I have to rollback to my own for now.

Reply:
Rolled back to ctovena/cortex-build:update-gogoproto-2294034c if a maintainer can pull/tag/push to quay that would be great.

Reply:
I take care of building the new image. Will post the updated tag as soon as I have it.

Reply:
I built a new image from this PR and published it:
`quay.io/cortexproject/build-image:update-gogoproto-9a1b26ac6`

Could you update `.circleci/config.yml` accordingly, please?